app/assets/javascripts/controllers/automation_manager/automation_manager_form_controller.js

```diff
@@ -8,79 +9,69 @@ ManageIQ.angular.app.controller('automationManagerFormController', ['$http', '$s
     log_password: '',
     log_verify: '',
   };
-  $scope.formId = automationManagerFormId;
-  $scope.afterGet = false;
-  $scope.validateClicked = miqService.validateWithAjax;
-  $scope.modelCopy = angular.copy( $scope.automationManagerModel );
-  $scope.model = 'automationManagerModel';
+  vm.formId = automationManagerFormId;
+  vm.afterGet = false;
+  vm.validateClicked = $scope.validateClicked = miqService.validateWithAjax;
+  vm.modelCopy = $scope.modelCopy = angular.copy(vm.automationManagerModel);
+  vm.model = 'automationManagerModel';
 
   ManageIQ.angular.scope = $scope;
 
-  if (automationManagerFormId === 'new') {
-    $scope.newRecord = true;
 
-    $scope.automationManagerModel.name = '';
-    $scope.automationManagerModel.url = '';
-    $scope.automationManagerModel.verify_ssl = false;
-    $scope.automationManagerModel.log_userid = '';
-    $scope.automationManagerModel.log_password = '';
-    $scope.automationManagerModel.log_verify = '';
+  if (automationManagerFormId === 'new') {
+    vm.newRecord = $scope.newRecord = true;
```

Comment:
Etiher `$scope.newRecord` is not needed here, or it is also needed on line 34, in the else case..